.github/workflows/pythonpackage.yml

```diff
@@ -28,4 +32,78 @@ jobs:
           coverage run -m unittest discover -s urwid -v
           coverage report
 #      - name: Coveralls
-#        uses: coverallsapp/github-action@v1
\ No newline at end of file
+#        uses: coverallsapp/github-action@v1
+
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    needs: [ Test ]
+    # build only on push: heavy job
+    if: github.event_name == 'push'
```

Comment:
This is heavy (see example run: https://github.com/penguinolog/urwid/actions/runs/5243263603 )